src/cairoWriter/writers/cairoContractWriter.ts

```diff
@@ -80,7 +81,7 @@ export class CairoContractWriter extends CairoASTNodeWriter {
       '@storage_var',
       'func WARP_NAMEGEN() -> (name: felt){',
       '}',
-      'func readId{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr : felt}(loc: felt) -> (val: felt){',
+      'fn readId(loc: felt) -> (val: felt){',
```

Comment:
This function is not fully converted, also it is fixed by the storage mappings branch. Please revert to it's original state

Reply:
The reason why is not fully converted is because it has not strightforward convertion. It should be handled as you said as part of the storage implementation. It have that old and invalid syntax just to mark that is an unadressed problem in this PR.

Reply:
But would you please revert it to it's original state since another PR will handle it?